src/main/java/io/codelex/arrays/practice/TicTacToe.java

```diff
@@ -0,0 +1,82 @@
+package io.codelex.arrays.practice;
+
+import java.util.Scanner;
+
+public class TicTacToe {
```

Comment:
ir opcija aizstāt jau iepriekš atzīmētu lauku, i guess its not intended :smile:

Reply:
Vai vari precizēt?

Reply:
ar šo biju domājis, ka aizliegt izvēlēties jau aizņemtu lauku, jo ja ieliku X `0 0`, nebija problēmu ielikt O turpat 😄